packages/select/src/common/listItemsProps.ts

```diff
@@ -112,6 +112,12 @@ export interface IListItemsProps<T> extends IProps {
      */
     resetOnSelect?: boolean;
 
+    /**
+     * Whether the active item should be scrolled into view when the active item is changed.
```

Comment:
explain the particular behavior of this prop -- it's not _always_ when the active item is changed.
it also only really applies when `props.activeItem` is controlled, right?